snooty/main.py

```diff
@@ -141,9 +140,11 @@ def on_update_metadata(
         self, prefix: List[str], field: Dict[str, SerializableType]
     ) -> None:
         property_name = "/".join(prefix)
-        self.client["snooty"]["metadata"].update_one(
-            {"_id": property_name}, {"$set": field}, upsert=True
-        )
+        # Write to Atlas if field is not an empty dictionary
+        if field:
+            self.client["snooty"]["metadata"].update_one(
```

Comment:
should we pull the database and collection name into a const? I realize that it's a bit late to make this recommendation so don't worry about it for now...